search/v2/geocoder.hpp

```diff
@@ -68,7 +69,7 @@ class Geocoder : public my::Cancellable
     Params();
 
     m2::RectD m_viewport;
-    m2::PointD m_position;
+    m2::PointD m_position;  ///< Default = {0, 0} as empty.
```

Comment:
But user position can really be (0, 0), right? It's a valid place on the Earth. If you need to specify an undefined state for m_position, please, introduce a boolean flag.